requirements.txt

```diff
@@ -3,7 +3,6 @@
 
 beautifulsoup4
 PyYAML==3.10
```

Comment:
I like alphabetica order. There's a bit of mess here, but I was planning of taking care of it separately.